cmd/utils/flags.go

```diff
@@ -81,1013 +81,1227 @@ func NewApp(gitCommit, usage string) *cli.App {
+		Name:   "extradata",
+		Usage:  "Block extra data set by the work (default = client version)",
+		EnvVar: "KLAYTN_EXTRADATA",
 	}
 
 	TxResendIntervalFlag = cli.Uint64Flag{
-		Name:  "txresend.interval",
-		Usage: "Set the transaction resend interval in seconds",
-		Value: uint64(cn.DefaultTxResendInterval),
+		Name:   "txresend.interval",
+		Usage:  "Set the transaction resend interval in seconds",
+		Value:  uint64(cn.DefaultTxResendInterval),
+		EnvVar: "KLAYTN_TXRESEND_INTERVAL",
 	}
 	TxResendCountFlag = cli.IntFlag{
-		Name:  "txresend.max-count",
-		Usage: "Set the max count of resending transactions",
-		Value: cn.DefaultMaxResendTxCount,
+		Name:   "txresend.max-count",
+		Usage:  "Set the max count of resending transactions",
+		Value:  cn.DefaultMaxResendTxCount,
+		EnvVar: "KLAYTN_TXRESEND_MAX_COUNT",
 	}
 	// TODO-Klaytn-RemoveLater Remove this flag when we are confident with the new transaction resend logic
 	TxResendUseLegacyFlag = cli.BoolFlag{
-		Name:  "txresend.use-legacy",
-		Usage: "Enable the legacy transaction resend logic (For testing only)",
+		Name:   "txresend.use-legacy",
+		Usage:  "Enable the legacy transaction resend logic (For testing only)",
+		EnvVar: "KLAYTN_TXRESEND_USE_LEGACY",
 	}
 	// Account settings
 	UnlockedAccountFlag = cli.StringFlag{
-		Name:  "unlock",
-		Usage: "Comma separated list of accounts to unlock",
-		Value: "",
+		Name:   "unlock",
+		Usage:  "Comma separated list of accounts to unlock",
+		Value:  "",
+		EnvVar: "KLAYTN_UNLOCK",
 	}
 	PasswordFileFlag = cli.StringFlag{
-		Name:  "password",
-		Usage: "Password file to use for non-interactive password input",
-		Value: "",
+		Name:   "password",
+		Usage:  "Password file to use for non-interactive password input",
+		Value:  "",
+		EnvVar: "KLAYTN_PASSWORD",
 	}
 
 	VMEnableDebugFlag = cli.BoolFlag{
-		Name:  "vmdebug",
-		Usage: "Record information useful for VM and contract debugging",
+		Name:   "vmdebug",
+		Usage:  "Record information useful for VM and contract debugging",
+		EnvVar: "KLAYTN_VMDEBUG",
 	}
 	VMLogTargetFlag = cli.IntFlag{
-		Name:  "vmlog",
-		Usage: "Set the output target of vmlog precompiled contract (0: no output, 1: file, 2: stdout, 3: both)",
-		Value: 0,
+		Name:   "vmlog",
+		Usage:  "Set the output target of vmlog precompiled contract (0: no output, 1: file, 2: stdout, 3: both)",
+		Value:  0,
+		EnvVar: "KLAYTN_VMLOG",
 	}
 	VMTraceInternalTxFlag = cli.BoolFlag{
-		Name:  "vm.internaltx",
-		Usage: "Collect internal transaction data while processing a block",
+		Name:   "vm.internaltx",
+		Usage:  "Collect internal transaction data while processing a block",
+		EnvVar: "KLAYTN_VM_INTERNALTX",
 	}
 
 	// Logging and debug settings
 	MetricsEnabledFlag = cli.BoolFlag{
-		Name:  metricutils.MetricsEnabledFlag,
-		Usage: "Enable metrics collection and reporting",
+		Name:   metricutils.MetricsEnabledFlag,
+		Usage:  "Enable metrics collection and reporting",
+		EnvVar: "KLAYTN_ETRICUTILS_METRICSENABLEDFLA",
```

Comment:
```suggestion
		EnvVar: "KLAYTN_METRICUTILS_METRICSENABLEDFLA",
```
Seems like `M` is missing?

Reply:
How about the below code? I think we must take care of a case when constant, `metricutils.MetricsEnabledFlag` is updated.

```suggestion
		EnvVar: "KLAYTN_ETRICUTILS_" + strings.ReplaceAll(strings.ReplaceAll(strings.ToUpper(metricutils.MetricsEnabledFlag), "-", "_"), ".", "_"),
```